orttraining/orttraining/training_ops/cuda/collective/nccl_common.cc

```diff
@@ -33,6 +33,7 @@ static Status CreateNcclCommunicator(MPI_Group* mpi_world_group,
                                      const training::WorkerGroupType worker_group_type,
                                      ncclComm_t* group_comm) {
   auto worker_group = training::DistributedRunContext::GetInstance().GetWorkerGroup(worker_group_type);
+  std::cout << "worker group " << worker_group_type << ", " << worker_group.ToString() << std::endl;
```

Comment:
Can we use the logging APIs here also

Reply:
Sure.
```
  LOGS_DEFAULT(INFO) << "worker group " << worker_group_type << ", " << worker_group.ToString();
```

---
In reply to: [532625282](https://github.com/microsoft/onnxruntime/pull/5815#discussion_r532625282) [](ancestors = 532625282)